docs/syntax.md

```diff
@@ -366,7 +366,7 @@ Use `range` to iterate over data, just like you would in Go, or how you would us
         {{.}}
     {{ end }}
 
-Jet provides built-in rangers for Go slices, arrays, maps, and channels. You can add your own by implementing the Ranger interface. TODO
+Jet provides built-in rangers for Go slices, arrays, maps, and channels. You can add your own by implementing the Ranger interface.
```

Comment:
do you intend to send another PR with more explanation about custom rangers? if not, we shouldn't remove this TODO yet, right?

Reply:
[Updated](https://github.com/CloudyKit/jet/compare/a56a8e1c4ae43c8fb5d16aee0cf71df931f79293..2544d9a974cd3b2003195a4b623781c4c510f43b) the docs with a section for the custom ranger, linking to the go doc page for the interface. I've also added a go-styled example (it will show up in the official v6 docs page after this is merged).